conf.py

```diff
@@ -51,29 +51,34 @@
 # If true, '()' will be appended to :func: etc. cross-reference text
 add_function_parentheses = True
 
+# list of folders that may contain Odoo sources
+odoo_sources_candidate_dirs = (Path('odoo'), Path('../odoo'))
+
 #=== Extensions configuration ===#
 
 # Add extensions directory to PYTHONPATH
 extension_dir = Path('extensions')
 sys.path.insert(0, str(extension_dir.absolute()))
 
 # Search for the directory of odoo sources to know whether autodoc should be used on the dev doc
```

Comment:
`odoo_sources_candidate_dirs` is only a local variable and not a Sphinx configuration var.
```suggestion
#=== Extensions configuration ===#

# Add extensions directory to PYTHONPATH
extension_dir = Path('extensions')
sys.path.insert(0, str(extension_dir.absolute()))

# Search for the directory of odoo sources to know whether autodoc should be used on the dev doc
odoo_sources_candidate_dirs = (Path('odoo'), Path('../odoo'))
```

Reply:
done :+1: